ClientPlugin/Settings/Generator.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ClientPlugin.Settings.Elements;
+using ClientPlugin.Settings.Layouts;
+using Sandbox.Graphics.GUI;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+
+namespace ClientPlugin.Settings
+{
+    internal class Generator
+    {
+        public readonly string Name;
+
+        private readonly List<Tuple<IElement, string, Func<object>, Action<object>>> Attributes;
```

Comment:
Introduce `class AttributeInfo` for the list item. Using tuples is usually a bad idea, because items are not named, which makes the code harder to read.

Reply:
Done in https://github.com/sepluginloader/PluginTemplate/pull/22/commits/90a395b9cfa475c505450901ef3f8634ffa35d95